scripts/fix/browser-order.ts

```diff
@@ -23,15 +23,17 @@ export const orderSupportBlock = (
   value: CompatStatement,
 ): CompatStatement => {
   if (key === '__compat') {
-    const support = Object.keys(value.support)
+    const support: CompatStatement['support'] = (
```

Comment:
Wouldn't it be better to do this instead?

```suggestion
    const support: SupportStatement = (
```

(Please feel free to ignore if this is just another one of those dumb TypeScript nits!)